packages/subgraph/tests/constants.ts

```diff
@@ -36,6 +36,10 @@ export const START_DATE = '1644851000';
 export const END_DATE = '1644852000';
 export const SNAPSHOT_BLOCK = '100';
 
+// Use 1 for testing as default value is anyways 0 
+// and test might succeed even though it shouldn't
+export const ALLOW_FAILURE_MAP = '1';
```

Comment:
`1` means that the first action can fail, is that correct? Won't this change the current test behavior?

Reply:
I only pass `1` in a test where actions are []`. We anyways don't test voting plugins as integration tests(also testing dao.execute results). So I only use `1` just to test the fact that it was stored correctly in a voting plugin and was emitted correctly.